certified-connectors/Cireson Service Manager Portal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2242 @@
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "webhookSettings": {
+                  "type": "object",
+                  "properties": {
+                    "Url": {
+                      "type": "string",
+                      "description": "Url",
+                      "title": "Url",
+                      "x-ms-notification-url": true,
+                      "x-ms-visibility": "internal"
+                    },
+                    "WorkItemClassType": {
+                      "type": "string",
+                      "description": "Comment Type",
+                      "title": "Comment Type",
+                      "enum": [
+                        "Analyst Comment",
+                        "EndUser Comment"
+                      ],
+                      "default": "Analyst Comment"
+                    }
+                  },
+                  "description": "webhookSettings",
+                  "required": [
+                    "Url",
+                    "WorkItemClassType"
+                  ]
+                }
+              },
+              "required": [
+                "webhookSettings"
+              ]
+            }
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {
+    "WorkItem_CreatedSRResponse": {
+      "type": "object",
+      "properties": {
+        "Id": {
+          "type": "string",
+          "description": "Id",
+          "title": "Id"
+        },
+        "Title": {
+          "type": "string",
+          "description": "Title",
+          "title": "Title"
+        },
+        "Description": {
+          "type": "string",
+          "description": "Description",
+          "title": "Description"
+        },
+        "Priority": {
+          "type": "object",
+          "description": "Priority",
+          "title": "Priority"
+        },
+        "Classification": {
+          "type": "object",
+          "description": "Classification",
+          "title": "Classification"
+        },
+        "Urgency": {
+          "type": "object",
+          "description": "Urgency",
+          "title": "Urgency"
+        },
+        "Impact": {
+          "type": "object",
+          "description": "Impact",
+          "title": "Impact"
+        },
+        "Source": {
+          "type": "object",
+          "description": "Source",
+          "title": "Source"
+        },
+        "SupportGroup": {
+          "type": "object",
+          "description": "Support Group",
+          "title": "Support Group"
+        },
+        "Status": {
+          "type": "object",
+          "description": "Status",
+          "title": "Status"
+        },
+        "AssignedWorkItem": {
+          "type": "object",
+          "description": "Assigned WorkItem",
+          "title": "AssignedWorkItem"

```

Comment:
@CiresonOps please make sure everything is grammatically correct and is free of typographical errors.